modules/core/src/test/java/org/locationtech/jts/simplify/TopologyPreservingSimplifierTest.java

```diff
@@ -141,36 +141,41 @@ public void testNoCollapseSmallSquare() throws Exception {
         "POLYGON ((0 0, 5 5, 5 0, 0 0))"
         );
   }
-  
+
   public void testPolygonRemoveEndpoint() throws Exception {
     checkTPS(
       "POLYGON ((220 180, 261 175, 380 220, 300 40, 140 30, 30 220, 176 176, 220 180))",
         40,
         "POLYGON ((30 220, 380 220, 300 40, 140 30, 30 220))"
         );
   }
-  
+
   public void testLinearRingRemoveEndpoint() throws Exception {
     checkTPS(
       "LINEARRING (220 180, 261 175, 380 220, 300 40, 140 30, 30 220, 176 176, 220 180)",
         40,
         "LINEARRING (30 220, 380 220, 300 40, 140 30, 30 220)"
         );
+    checkTPS(
+      "LINESTRING (220 180, 261 175, 380 220, 300 40, 140 30, 30 220, 176 176, 220 180)",
```

Comment:
This test is intended to test behaviour on `LINEARRING` geometries.   A `LINESTRING` geometry should be in a separate test.